database.go

```diff
@@ -1049,6 +1049,35 @@ func (rp *RetentionPolicy) shardGroupByID(shardID uint64) *ShardGroup {
 	return nil
 }
 
+// dropMeasurement will remove a measurement from the index.
```

Comment:
I'd like this comment to be very precise. The "index" in our system is the in-RAM representation of the what is in the Metastore (which series are in which measurements, etc). The raw data is something else.

So what does this function affect? The in-RAM index, the Metastore on disk, and the actual raw series data?